packages/esm-patient-immunizations-app/package.json

```
@@ -37,13 +37,15 @@
     "url": "https://github.com/openmrs/openmrs-esm-patient-chart/issues"
   },
   "dependencies": {
-    "@carbon/react": "^1.12.0",
-    "@openmrs/esm-patient-common-lib": "^6.1.0",
-    "lodash-es": "^4.17.21"
+    "@hookform/resolvers": "^3.3.1",
+    "lodash-es": "^4.17.21",
+    "react-hook-form": "^7.46.2",
+    "zod": "^3.22.2"
   },
   "peerDependencies": {
+    "@carbon/react": "1.x",
     "@openmrs/esm-framework": "5.x",
-    "@openmrs/esm-patient-common-lib": "5.x",
+    "@openmrs/esm-patient-common-lib": "*",
```

I think this is what we're using elsewhere (after a PR from Dennis)

```suggestion
    "@openmrs/esm-patient-common-lib": "6.x",
```